include/pelz_uri_helpers.h

```diff
@@ -55,7 +55,7 @@ extern "C"
  *
  * @return 0 on success, 1 on error
  */
-  int get_pelz_uri_port(UriUriA uri, int *port);
+  int get_pelz_uri_port(UriUriA uri, charbuf * port);
```

Comment:
Why did you switch the port from an int to a charbuf?

Reply:
The port value used by enclave_retrieve_key is a char string with a length value and not an int.